third_party/gtp_ovs/ovs-gtp-patches/2.15/build.sh

```diff
@@ -5,6 +5,7 @@ OVS_VER='2.15'
 DIR="ovs-build"
 DEST=$1
 
+sudo apt update
```

Comment:
Could we add a licence header for this script?

Reply:
Optional: Add `set -euox pipefail` instead of `set -ex` at the top and make sure that `shellcheck third_party/gtp_ovs/ovs-gtp-patches/2.15/build.sh` is clear.